app/views/projects/blob/new.html.haml

```diff
@@ -1,5 +1,11 @@
-- page_title "New File", @ref
-%h3.page-title New file
+%h3.page-title<
+  Create new file or 
+  %a.button{href: '#modal-upload-blob', 'data-target' => '#modal-upload-blob', 'data-toggle' => 'modal'}> &nbsp;upload existing one
```

Comment:
I changed the space highlight, it looks like this now
![spacehightlight](https://cloud.githubusercontent.com/assets/7370334/9838217/fa8bec6c-5a57-11e5-895e-0df2fc6afcae.png)